docs/pages/develop/user-interface/app-icons.mdx

```diff
@@ -28,18 +28,15 @@ For an in-detail walkthrough, see the video below:
 
 ### Android
 
-{/* TODO: (@aman) Remove comments below when monochrome is released for the next SDK. */}
+Further customization of the Android icon is possible using the [`android.adaptiveIcon`](/versions/latest/config/app/#adaptiveicon) property, which will override both of the previously mentioned settings.
 
-Further customization of the Android icon is possible using the [`android.adaptiveIcon`](versions/latest/config/app/#adaptiveicon) property, which will override both of the previously mentioned settings.
-
-The Android Adaptive Icon is formed from two separate layers &mdash; a foreground image and a background color or image. This allows the OS to mask the icon into different shapes and also supports visual effects.
-{/* - For Android 13 and later, the OS supports a monochrome foreground image layer which will use a background color taken from the device's theme. */}
+The Android Adaptive Icon is formed from two separate layers &mdash; a foreground image and a background color or image. This allows the OS to mask the icon into different shapes and also supports visual effects. For Android 13 and later, the OS supports a monochrome foreground image layer which will use a background color taken from the device's theme.
```

Comment:
Let's call this what Google calls it, which is: 'themed app icon support'. Also, the term 'background color' isn't entirely accurate, it uses wallpaper and theme colors.